### PR TITLE
v0.4.12

### PR DESCRIPTION
**Pull request by benpatterson**
TE-1218

### CODE REVIEW

**Comment by benpatterson** [comment]
@jzoldak @cahrens @dianakhuang I can rebase this PR once the XSS work lands. LMK if you have feedback on these changes.

File: CHANGELOG [hunk 1]
```diff
@@ -1,5 +1,17 @@
+v0.4.12 (3/22/16)
```
Comment (jzoldak) [inline CHANGELOG]: I thought we were talking about making this 0.5.0?
We bumped up to 0.4.0 when we added the visual diff and a11y testing features.

**Comment by dianakhuang** [comment]
@benpatterson The XSS PR just merged: https://github.com/edx/bok-choy/pull/139

File: CHANGELOG (hunk 1 above)
Comment (benpatterson) [inline CHANGELOG]: Och you are right. Let me fix that.

**Comment by benpatterson** [comment]
I'm going to close this and create a new one since I had given the branch name the name of the wrong release version.

**Pull request closed without merging**